aksel.nav.no/website/pages/eksempler/datepicker/disabled-days.tsx

```diff
@@ -1,16 +1,16 @@
 import { DatePicker } from "@navikt/ds-react";
 import { withDsExample } from "@/web/examples/withDsExample";
 
-const Example = () => {
-  const disabledDays = [
-    new Date("Oct 10 2022"),
-    { from: new Date("Oct 17 2022"), to: new Date("Nov 10 2022") },
-    (date) => isFriday(date),
-  ];
+const disabledDays = [
+  new Date("Oct 14 2024"),
+  { from: new Date("Oct 21 2024"), to: new Date("Nov 10 2024") },
```

Comment:
I thought it might be best to keep this one hard coded. We use the `today` prop to make sure the picker always opens on the "correct" month. Also, the first disabled day might be a Friday on some years (which we want to avoid since we already disable Fridays dynamically).